Buildings/Fluid/HeatExchangers/BaseClasses/VariableSpeedThermalWheels/BaseClasses/Characteristics/heatExchangerEffectivenessParameters.mo

```diff
@@ -0,0 +1,27 @@
+within Buildings.Fluid.HeatExchangers.BaseClasses.VariableSpeedThermalWheels.BaseClasses.Characteristics;
+record heatExchangerEffectivenessParameters
+  "Record for heat exchanger effectiveness correction vs. wheel speed ratio"
+  extends Modelica.Icons.Record;
+  parameter Real uSpe[:](each min=0)
+    "Wheel speed ratio";
+  parameter Real epsCor[:](each max=1)
+    "Correction of the heat exchange effectiveness under a given speed ratio";
```

Comment:
`Correction of the heat exchange effectiveness at a given speed ratio`

Reply:
Update the model doc as suggested.